SwiftWisdom/Core/StandardLibrary/String/String+Trimmed.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  String+Trimmed.swift
+//  SwiftWisdom
+//
+//  Created by Maya Saxena on 6/13/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    func ip_trimmed(toLength length: Int) -> String {
+        if ip_length > length {
+            return substring(to: index(startIndex, offsetBy: length))
```

Comment:
Swift 3 has a `prefix` method that is slightly shorter.  See 
"The New Substring Type" in https://www.swiftbysundell.com/posts/exploring-the-new-string-api-in-swift-4 for Swift 3 *and* Swift 4 ways to do this.